galley/pkg/config/analysis/local/analyze.go

```diff
@@ -77,9 +77,10 @@ func (sa *SourceAnalyzer) Analyze(cancel chan struct{}) (diag.Messages, error) {
 }
 
 // AddFileKubeSource adds a source based on the specified k8s yaml files to the current SourceAnalyzer
-func (sa *SourceAnalyzer) AddFileKubeSource(files []string) error {
+func (sa *SourceAnalyzer) AddFileKubeSource(files []string, defaultNs string) error {
 	src := inmemory.NewKubeSource(sa.m.KubeSource().Resources())
 
+	src.SetDefaultNamespace(defaultNs)
```

Comment:
Extremely Minor Nit: move the space between NewKubeSource and setDefaultNamespace to be between setDefaultNamespace and the forloop. The new & set or more related operations.

Reply:
Done.